mne/preprocessing/ica.py

```diff
@@ -1215,28 +1215,29 @@ def _pick_sources(self, data, include, exclude):
         if self.pca_mean_ is not None:
             data -= self.pca_mean_[:, None]
 
-        pca_data = fast_dot(self.pca_components_, data)
-        # Apply unmixing to low dimension PCA
-        sources = fast_dot(self.unmixing_matrix_, pca_data[:n_components])
-
+        sel_keep = np.arange(n_components)
```

Comment:
slice(None) should be faster (will be view) since you don't modify it later and don't do anything inplace